README-ckad.md

```diff
@@ -0,0 +1,105 @@
+
+# Certified Kubernetes Application Developer (CKAD)
+A  curated list of resources that will help you to face your Kubernetes certification. Based in the newest policy about 'Resources Allowed', current list will give preference
+to those materials coming from the allowed sources [kubernetes.io](kubernetes.io). Others like Youtube videos, blog resources could be referenced in expectional case due to I still found them useful in my k8s learning journey.
```

Comment:
The last sentence can be rephrased as : 

```
Other resources like YouTube videos, blogs etc could be referenced in the exceptional case since I still found them useful in my k8s learning journey.
```